Tests/Opc.Ua.Server.Tests/CreateSessionApplicationUriValidationTests.cs

```diff
@@ -323,7 +323,7 @@ public void CreateSessionWithMultipleUrisNoneMatchThrows(NodeId certificateType)
                 // Create and open session with retry logic for transient errors
                 var sessionFactory = new DefaultSessionFactory(telemetry);
                 const int maxAttempts = 5;
-                const int delayMs = 1000;
+                const int delayMs = 10000;
```

Comment:
Ca you add a comment why this fixes the flakiness?   And can regular code with this setting at 1 sec run into issues?